docs/swagger.yaml

```diff
@@ -372,6 +376,42 @@ components:
       required:
         - orderLines
         - bottomLine
+      x-examples:
+        example-1:
+          orderLines:
+            - name: Vipps socks
+              id: '1234567890'
+              totalAmount: 1000
+              totalAmountExcludingTax: 800
+              totalTaxAmount: 250
+              taxPercentage: 25
+              unitInfo:
+                unitPrice: 0
+                quantity: '0.822'
+                quantityUnit: PCS
+              discount: 0
+              productUrl: 'https://www.vipps.no/store/socks'
+              isReturn: false
+              isShipping: false
+          bottomLine:
+            currency: NOK
+            tipAmount: 0
```

Comment:
remove the deprecated lines from the example.
You could also update the description on tipAmount and say "use paymentSources" instead :)

Reply:
tip is not part of the paymentsources?

Reply:
No, tip isnt part of the paymentSources, but it shouldnt be in the example either. All fields that are deprecated should be hidden from the example